kuma/javascript/src/router.jsx

```diff
@@ -66,7 +66,10 @@ export default function Router({
         params: ?RouteParams,
         // Data fetched asynchronously by the Route.fetch() function.
         // This value will be passed to the component as the data prop.
-        data: ?RouteData
+        data: ?RouteData,
+        // The loading state of the page. Defaults to `true`, as it should
+        // always appear until the component finishes rendering
```

Comment:
That line isn't correct. The "loading" state has nothing to do with rendering. It has to do with fetching.